modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.js

```diff
@@ -14,6 +14,64 @@
 
 Liferay = window.Liferay || {};
 
+Liferay.Webpack = {
```

Comment:
This feels like a good setup for race conditions 😂

Reply:
TBH, I'm not sure where to put it. I just put it there for the PoC, but we should probably move it to its own place (a file inside frontend-js-web?) :thinking:

Reply:
Given that it replaces the AMD loader, I thought one of the places near it would be best, but I'm not sure if it is the best option.